library/qtest/src/Operations.qs

```diff
@@ -12,13 +12,13 @@ import Std.Arrays.Mapped, Std.Arrays.All;
 /// test results instead of printing out to output.
 ///
 /// # Input
-/// Takes a list of test cases. A test case is a tuple of `(String, () -> T, 'T)`, where
```

Comment:
This takes 5 values as input, am I correct. This is related to my confused of `Signed Int` library tests passing in which test case is tuple of 5 values.

Reply:
Ah I see what you mean. Updated in d283223